docs/source/developer/tests.rst

```diff
@@ -117,27 +126,170 @@ test is run while `record_flight_data` is in place.
 When the test is completed, change to using `replay_flight_data`:
 
   .. code-block:: python
-    :emphasize-lines: 2,3
+    :emphasize-lines: 2
 
-    def test_example(self):
-        session_factory = self.replay_flight_data(
-            'test_example')
+    def test_example(self, test):
+        session_factory = test.replay_flight_data('test_example')
 
         ...
 
 Now when the test is run it will use the data previously recorded and will not
-contact AWS.  When committing your test, don't forget to include the 
+contact AWS.  When committing your test, don't forget to include the
 `tests/data/placebo/test_example` directory!
 
 Note: if it's necessary to delay CLI calls due to delays in the time it takes
 for an attribute on a resource to be reflected in an API call or any other reason,
-use ``self.recording`` to only sleep when recording json like so:
+use ``test.recording`` to only sleep when recording json like so:
 
   .. code-block:: python
 
     import time
 
     ...
 
-    if self.recording:
+    if test.recording:
       time.sleep(10)
+
+Writing Terraform Bootstrapped Tests for AWS Resources
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Note: These instructions are only for recording data in AWS. For Azure, GCP, or
```

Comment:
can we drop this comment, we'll add examples of the other providers.

Reply:
Sounds good!